pkg/security/secl/rules/policy_dir.go

```diff
@@ -103,6 +103,11 @@ func (p *PoliciesDirProvider) LoadPolicies(macroFilters []MacroFilter, ruleFilte
 		if err != nil {
 			errs = multierror.Append(errs, err)
 		}
+
+		if policy == nil {
```

Comment:
shouldn't we just continue between line 104 and 105

Reply:
oh yeah no really since we want to recover from errors we can have both policy != nil and err != nil